airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -206,6 +206,15 @@
                     }
                   }
                 ]
+              },
+              "file_buffer_size": {
+                "title": "File Buffer Count",
+                "type": "integer",
+                "minimum": 15,
```

Comment:
the new test checks that this value can be set as low as `10`, but the spec says that minimum us `15`. Should the two minimums be the same?

Reply:
Good catch. Yes, the minimums should be the same. I'll add on a test and change that ensures at minimum there are `15` file buffers